packages/suite/src/components/suite/modals/AddAccount/components/AddCoinJoinAccountButton.tsx

```diff
@@ -88,6 +81,7 @@ export const AddCoinJoinAccountButton = ({ network }: AddCoinJoinAccountProps) =
                 isDisabled ? <Translation id="MODAL_ADD_ACCOUNT_LIMIT_EXCEEDED" /> : null
             }
             handleClick={onCreateCoinjoinAccountClick}
+            isLoading={isLoading}
```

Comment:
Thanks! That loading effect was really needed. :+1:

Reply:
Your idea 😉 still not perfect, there is a period with modal already closed but CoinJoin account still not visible...